sdcm/cluster.py

```diff
@@ -3867,7 +3867,7 @@ def update_scylla_packages(node, _queue):
                 check_package_suites_distro(node, 'deb')
 
                 def list_installed_packages():
-                    node.scylla_packages_installed()
+                    node.scylla_packages_installed
```

Comment:
instead of this, remove this 2 lines, and in the calls below, just use `node.scylla_packages_installed`

Reply:
thanks, fixed now.